### PR TITLE
Job coloring rework

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR 
Rework Job, ID card and Encryption keys so it is a bit more fitting overall. Probably needs discussion for certain colors, icon choices etc.
For example the Valet would make sense to put under Cargo/Logistics colors but eh.
They are still all under HoP supervision.

Is this a big list for what amounts to tiny sprite changes? Yes, I wanted to be thorough with this for some reason.

Trying to follow the previously set color scheme of the "Service" departement jobs, they are split into Logistics/Cargo(Brown), Catering(Green), Civilian(Grey) and "Entertainment"(Pink).

- Job icons changes
  - Assistant is a better letter A.
  - Roboticist now uses Engineering departement colors even if unused.
  - Valet now reuses the sprite for the Service encryption key as it fits relatively well.
  - Gladiator gets a new sprite in the form a sword and the same colors as the prisoner icon.
  - Martial Artist gets a new sprite in the form of a karate chop hand or something like that.
  - Reporter changes into Civilian grey.
  - Cyborg and MedicalCyborg gets job icons.
  - Service job icons are now greener, which makes it easier on the eyes and identical to their ID card icons

- Encryption key changes
  - Renames the "science" encryption keys to "epistemic" encryption keys
  - Salvage, QM and Mysta encryption keys sprites uses their correct color even if unused.
  - Roboticist encryption card now uses Engineering department colors even if unused.
  - The Common encryption key now uses the Assistant job icon sprite.

- ID Card changes
  - Salvage now uses the Miner sprites which has been recoloured to the Engineering departement
  - Roboticist now uses Engineering departement colors even if unused.
  - Assistant now has a proud A instead of a P.
  - Valet uses the new job icon that somewhat looks like someone holding a plate.
  - Chaplain, Mantis, Mystagogue and Acolyte now uses Epistemic departement colors, white and black.
  - Cataloguer now uses the Librarian sprite which has been recolored to the Epistemic departement colors.
  - Martial Artist gets a new ID icon similar to their new job icon.
  - Gladiator now uses the gladiator job icon and the same colors as the prisoner icon.
  - Boxer, Martial Artist, Musician get a pink icon and trim
  - Lawyer, Reporter and Janitor have a grey icon and trim
  
- Job Spawner changes using the wiki icons I made previously.
  - Add sprites for Gladiator, Martial Artist, Boxer, Valet, Mail Carrier and Musician spawners.
  - Replaces the spawner sprites for; Scientist, Lawyer, Reporter, Librarian, Paramedic, RD, Janitor to better fit our current uses of them and how they look.
  - Give the Cyborg and MedicalCyborg sprites.
  
- Various changes
  - Musician supervisor is now the HoP instead of "Anyone that hires you"
  - Using the Locale for the supervisors instead of plain text (mostly mystagogue)

**Media**
**Changes to the encryption key sprites**
Left to right; Common, MedEpi, Epi, Roboticist, Salvage, QM, Mysta
encryptChange
In game
ss (2023-04-16 at 06 25 31)
<br>

**Changes to the job icons sprites**
Left to right; Assistant, Roboticist, Valet, Gladiator, Martial Artist, Reporter, Botanist, Bartender, Zookeeper, Chef
jobchange
In game
ss+(2023-04-16+at+09 06 36)
Cyborg job icons
ss (2023-04-16 at 06 49 04)
<br>

**Changes to the ID card icons sprites**
Left to right; Assistant, Salvage, Roboticist, Valet, Mantis, Cataloguer, Chaplain, Acolyte, Mysta, Boxer, Gladiator, Martial Artist, Janitor, Reporter, Lawyer, Musician
idChange
Service sub department visualized; Civilian, Catering, Cargo and Entertainment
ss (2023-04-16 at 06 25 36)
Epistemics
ss (2023-04-16 at 06 25 38)
Salvage Tech gets their own ID too
ss (2023-04-16 at 06 25 39)

Map job spawners
ss (2023-04-16 at 06 48 47)

**Changelog**

:cl:
- tweak: Tweaked job and ID card icons to be more in line with their department.
- add: Added Cyborgs job icons.

### CODE REVIEW

**Comment by Vordenburg** [comment]
This is looking really good.

What's the mystagogue encryption key sprite about? You said the salvage and QM are unused, but is the mystagogue also unused?

**Comment by FluffiestFloof** [comment]
> This is looking really good.
> 
> What's the mystagogue encryption key sprite about? You said the salvage and QM are unused, but is the mystagogue also unused?


Yep pretty much unused.
I think it's due to upstream going for more of a "department" encryption keys instead of "role" encryption keys.

**Comment by Vordenburg** [comment]
Criticisms I can think of at the moment:
- The mime icon says more Space Invaders than mime to me.
- Is there a better icon we can have for the valet? At first glance, I thought the reporter was the valet, because it looked like a bell (bellhop).
- The Epi encryption key looks like an H but I can see how it's supposed to be an !.
- Do the `A`s look better if you use 2 pixels of spacing instead of 1 between the top and middle line?

**Comment by FluffiestFloof** [comment]
> Criticisms I can think of at the moment:
> 
>     * The mime icon says more Space Invaders than mime to me.
> 
>     * Is there a better icon we can have for the valet? At first glance, I thought the reporter was the valet, because it looked like a bell (bellhop).
> 
>     * The Epi encryption key looks like an H but I can see how it's supposed to be an !.
> 
>     * Do the `A`s look better if you use 2 pixels of spacing instead of 1 between the top and middle line?

* The mime icon is (almost, it was recolored pink at one point) unchanged from upstream and I don't see what else it could be personally. 
* Reporter icon is from upstream, just less green now. We could go back to the green A for valet, I thought using the service icon from the encryption key would look better for the valet. Or actually thinking about it, a V for Valet.
* I just followed the pattern of using the job icon for the encryption key icon. Might be because epistemic are the only one not using white for the actual icon and instead uses black.
* It looks different;
ss (2023-04-17 at 05 36 39)
ss (2023-04-17 at 05 36 55)

**Comment by Vordenburg** [comment]
2 pixel spacing A looks more balanced to me.

**Comment by FluffiestFloof** [comment]
For the Valet we could ignore the service icon (which looks like someone holding a plate or something) and just do a V
ss (2023-04-17 at 05 54 08)
ss (2023-04-17 at 05 54 48)

File: Resources/Prototypes/Catalog/Fills/Boxes/general.yml [hunk 1]
```diff
@@ -367,7 +367,7 @@
         amount: 6
 
 - type: entity
-  name: science encryption key box
+  name: epistemic encryption key box
```
Comment (Vordenburg) [inline Resources/Prototypes/Catalog/Fills/Boxes/general.yml]:
```suggestion
  name: epistemics encryption key box
```

File: Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml [hunk 2]
```diff
@@ -142,8 +142,8 @@
 - type: entity
   parent: EncryptionKey
   id: EncryptionKeyScience
-  name: science encryption key
-  description: An encryption key used by scientists. Maybe it is plasmaproof?
+  name: epistemic encryption key
```
Comment (Vordenburg) [inline Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml]:
```suggestion
  name: epistemics encryption key
```

**Review by Vordenburg — CHANGES_REQUESTED** [review]
I think we're ready to go on this after these couple lines are made consistent with the epistemics headset.

**Review by Vordenburg — APPROVED** [review]

**Pull request merged**